src/assets/icons/index.js

```diff
@@ -82,7 +82,14 @@ export const Google = () => (
 
 export const Twitter = () => {
   return (
-    <div style={{ backgroundColor: '#03A9F4', borderRadius: 25, display: 'inline', padding: 6 }}>
+    <div
+      style={{
```

Comment:
Use a class instead.

Reply:
Why should I ?,  
 it's a  dumb component and we are simply exporting SVG